parsers/sql.c

```diff
@@ -478,6 +480,21 @@ static void parseIdentifier (vString *const string, const int firstChar)
 		ungetcToInputFile (c);		/* unget non-identifier character */
 }
 
+static tokenType parseDollarQuote (vString *const string, const int firstChar)
+{
+	int c = firstChar;
+	do
+	{
+		vStringPut (string, c);
+		c = getcFromInputFile ();
+		if (c == EOF)
+			return TOKEN_EOF;
```

Comment:
Probably not very important, but I would rather simply cut off the end and still return the current token, and let the next one return `TOKEN_EOF`, like string does.  But that probably doesn't change anything.